lucene/CHANGES.txt

```diff
@@ -84,6 +84,8 @@ API Changes
 
 Improvements
 
+* LUCENE-9618: Do not call IntervalIterator.nextInterval after NO_MORE_DOCS is returned
```

Comment:
You can add your name to the end of this `CHANGES` entry (in parens)?

Reply:
Thanks for reminding me! Done